Doc/library/importlib.rst

```diff
@@ -1719,6 +1719,27 @@ To import a Python source file directly, use the following recipe
   spec.loader.exec_module(module)
 
 
+Implementing lazy imports
+'''''''''''''''''''''''''
+
+The example below shows how to implement lazy imports::
```

Comment:
This is one code block with two styles (console and Python), does it render correctly?

Reply:
Went with console.